include/_kernel.h

```diff
@@ -55,6 +55,10 @@
 /* must come after _enable_all_exts.h b/c of pocl_types.h*/
 #include "_kernel_c.h"
 
+#define __kernel_exec(X, typen)                                               \
```

Comment:
This is the weirdo macro from the specs. Please add a comment that this is from the specs,
perhaps a pointer to the html page :)